frontend/src/pages/Registration/Registration.tsx

```diff
@@ -50,9 +51,7 @@ export const Registration: FC = () => {
         autoCorrect={false}
       />
       <FormButton buttonTitle="Register" onPress={() => null} />
-      <Button title="Login" onPress={() => {() =>
-            this.props.navigation.navigate('Login')}}
```

Comment:
I can see this is from an example that uses a class-based component. `this` refers to the component object (with functional components we just pass props as parameters of the function so we don't need `this.`. Also if we destructure the props (`...componentName: FC = ({navigation}) => ...`) then we don't need to use `props.` either